### PR TITLE
Add min_pool_size option

### PR DESCRIPTION
**Pull request by Jacobkg**
Do you know if its a problem to add a field to the database like this? Is there some sort of migration that is necessary or anything?

@mgbatchelor @awebneck @awesinine @dferrantelli

### CODE REVIEW

**Comment by Jacobkg** [comment]
[Set a minimum pgbouncer pool size of 100](https://trello.com/c/gIRh2wIJ/82-set-a-minimum-pgbouncer-pool-size-of-100)

File: templates/pgbouncer.ini.erb [hunk 1]
```diff
@@ -125,6 +125,13 @@ max_client_conn = <%= @max_client_conn %>
 ; max clients you want to handle at any moment
 default_pool_size = <%= @default_pool_size %>
 
+;; Minimum number of server connections to keep in pool.
+<%- if @min_pool_size -%>
+min_pool_size = <%= @min_pool_size %>
+<%- else -%>
+;min_pool_size = 0
```
Comment (mgbatchelor) [inline templates/pgbouncer.ini.erb]: @Jacobkg ah I see, commented out :)

Comment (Jacobkg) [inline templates/pgbouncer.ini.erb]: Just cargo-culting what I saw elsewhere in the file

**Comment by dferrantelli** [comment]
LGTM

**Pull request merged**